### ISSUE TITLE
Поправить точку отсчёта при уведомлении об отзыве эгиды

### ISSUE THREAD

**Issue by nfarukshin** [maii-chgk/documents#25]
Сейчас точка отсчёта — решение об отзыве, а должно быть по идее сам отзыв

Оригинал: 
1.3.2.1.3. О том, что эгида отозвана, ТГ должна уведомить оргкомитет турнира не более чем через 3 суток после решения об отзыве.

**Comment by nfarukshin** [comment]
Предлагаемая поправка: 

1.3.2.1.3. [ТГ](https://www.maii.li/p/aegis-rating#tg) должна уведомить оргкомитет турнира об отзыве эгиды. Это должно произойти не более чем через 3 суток после того, как эгида отозвана.

**Comment by olek3** [comment]
> Сейчас точка отсчёта — решение об отзыве, а должно быть по идее сам отзыв

А чем они различаются? где расписана процедура отзыва эгиды? Там есть какой-то срок после решения, когда отзыв вступает в силу?

**Comment by nfarukshin** [comment]
Решение — это либо завершение голосования в чате (его сложно датировать), либо публикация текста решения (странно, а почему не предыдущий вариант?). Между решением и непосредственно отзывом может пройти какое-то время (у нас тут волонтёрская работа, не все решения реализуются сразу же). 

Регламентировать всё это можно, но смысла особого я не вижу (проще привязаться к моменту отзыва). Процедуры особой тоже не прописано, и смысла особого в этом тоже я не вижу.

**Comment by nfarukshin** [comment]
>как же привязаться к тому, чего нету?
Давайте непосредственно к делу, а то с такими темпами обсуждения мы ещё пару лет об этом будем говорить :)

Вы хотите сказать, что лучше также добавить определение момента отзыва эгиды? Или в чём цель этих вопросов?

**Comment by olek3** [comment]
> Вы хотите сказать, что лучше также добавить определение момента отзыва эгиды?
да, именно. Я просто думал, что она где-то определена, а я пропустил и мне покажут, где. Если нет определения, то оно нужно.

**Comment by nfarukshin** [comment]
Добавил пунктом 1.3.2.1.4
